src/rahul/ProgrammingTest2/Exam2.java

```diff
@@ -0,0 +1,40 @@
+/*Program 2 : Write a program to rearrange string as per below example.
+
+String input = "TeCHNoc33r44editS";
+output : 14eocreditTCHNS*/
+
+package rahul.ProgrammingTest2;
+
+public class Exam2 {
+	String outputDigit = "";
+	String outputUppercase = "";
+	String outputLowercase = "";
+	int sum;
+
+	void printSumString(String input) {
+		for (int index = 0; index < input.length(); index++) {
+			if (Character.isDigit(input.charAt(index))) {
```

Comment:
You can write char ch = input.charAt(index) outside all conditions, it will optimize code

Reply:
Sure, noted!